docs/contributing/getting-started.md

```diff
@@ -0,0 +1,68 @@
+This guide covers how to start contributing to Sedna 😄
+
+## 1. Familiarize Yourself With Contributing to Sedna
+
+### Setup GitHub Account
+
+Sedna is developed on [GitHub][github] and will require
+an account for contribution.
+
+### Check The Kubernetes Contributor Guides
+
+You may come back to this later, but we highly recommend reading these:
+
+- [Kubernetes Contributor Guide](https://git.k8s.io/community/contributors/guide) 
+  - Main contributor documentation, or you can just jump directly to the [contributing section](https://git.k8s.io/community/contributors/guide#contributing)
```

Comment:
These two links seem to jump to the same page, why do we repeat it?

Reply:
this guide is large and the second link located at the specified contributing section could be helpful